net/udpxy/Makefile

```diff
@@ -1,19 +1,14 @@
-#
-# Copyright (C) 2006-2016 OpenWrt.org
-#
-# This is free software, licensed under the GNU General Public License v2.
-# See /LICENSE for more information.
-#
-
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=udpxy
-PKG_VERSION:=1.0-25.1
+PKG_REV:=23b434374d76e5de74138d44cbb8bda1b32dcfe0
+PKG_VERSION:=2024.03.03
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://codeload.github.com/pcherenkov/udpxy/tar.gz/$(PKG_VERSION)?
-PKG_HASH:=a1a16e60895c6b2fd151321db47f5d5373843116f1b98ed9749e6c25a6c44497
+PKG_SOURCE_URL:=https://codeload.github.com/pcherenkov/udpxy/tar.gz/$(PKG_REV)?
```

Comment:
I'd switch to local tarballs instead. Avoids having to override BUILD_DIR.

Reply:
Done in 229deb7a37222277cb50b458f8a21f78e34ba766